components/RuiExpandCollapse/src/RuiExpandCollapse.ts

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright Deloitte Digital 2019
+ *
+ * This source code is licensed under the BSD-3-Clause license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, property, CSSResultArray, TemplateResult} from 'lit-element';
+import { variables, layout } from './RuiExpandCollapse.css'
+
+// Update to include any possible type a value
+// can take, currenlty only have boolean open property
+type RuiExpandCollapsePropertyType = boolean;
```

Comment:
I'm a little confused here. This doesn't look right.

I would like to research this a bit more but will not block merge.